setup.py

```diff
@@ -58,8 +58,10 @@ def run(self):
 
         if self.type in ['unit', 'all']:
             cmd.append('python/Ganga/new_tests/Unit')
-            cmd.append('python/Ganga/Core')
-            cmd.append('python/Ganga/Runtime')
+            ## Ganga/Core finds zero tests
+            #cmd.append('python/Ganga/Core')
```

Comment:
Why are we putting tests in Ganga/Core in this case?
I was under the impression that we were loading all tests from a dedicated directory with the tests isolated from the codebase.
Or have I misunderstood something?